src/main/java/org/opendevstack/provision/services/openshift/OpenshiftService.java

```diff
@@ -0,0 +1,99 @@
+/*
+ * Copyright 2017-2019 the original author or authors.
```

Comment:
I guess another nitpick 😊 but I'm wondering if these copyright mantras are really required? If not, I see them mostly as code noise.

Reply:
Yeah, good point, but it was introduced in the past, so I think it is better to keep adding it instead of having some classes with the copyright and some without it. I'll inform myself about the convention in ods about this.